examples/az_cli.py

```diff
@@ -74,3 +75,81 @@ def CreateDiskCopy(args: 'argparse.Namespace') -> None:
                                        src_profile=args.src_profile,
                                        dst_profile=args.dst_profile)
   logger.info('Done! Disk {0:s} successfully created.'.format(disk_copy.name))
+
+
+def StartAnalysisVm(args: 'argparse.Namespace') -> None:
```

Comment:
It is better to have the vm object returned here, allows for further operations and checking the creation status without intializing a vm using its name. It might be a problem if the creation failed because  the user must first verify the VM exist before using it to avoid problems unhandled creation errors. when I look to the gcp one, it does return:
Returns:
    Tuple(GoogleComputeInstance, bool): A tuple with a virtual machine object
        and a boolean indicating if the virtual machine was created or not.